build_thirdparty.sh

```diff
@@ -39,3 +39,32 @@ set -x
 
 # shellcheck disable=SC2086
 python3 "$YB_THIRDPARTY_DIR/python/yugabyte_db_thirdparty/yb_build_thirdparty_main.py" "$@"
+
+rc=$?
+
+set +xue
```

Comment:
Let's not switch to the `set +ue` mode (allowing undefined identifiers, ignoring errors). This is a much more error prone way to write scripts.